src/actions/app.js

```diff
@@ -66,6 +50,41 @@ export const { removeReaderFromDataset } = createActions({
   }),
 });
 
+export const { getStudies } = createActions({
+  [ActionTypes.GET_STUDIES_SUCCESS]: studies => studies,
+  [ActionTypes.GET_STUDIES]: (limit, offset, sort, direction, searchString) => ({
+    limit,
+    offset,
+    sort,
+    direction,
+    searchString,
+  }),
+});
+
+export const { getStudyById } = createActions({
+  [ActionTypes.GET_STUDY_BY_ID]: study => study,
+  [ActionTypes.GET_STUDY_BY_ID_SUCCESS]: study => study,
+});
+
+export const { getStudyPolicy } = createActions({
+  [ActionTypes.GET_STUDY_POLICY]: policy => policy,
+  [ActionTypes.GET_STUDY_POLICY_SUCCESS]: policy => policy,
+});
+
+export const { addCustodianToStudy } = createActions({
+  [ActionTypes.ADD_CUSTODIAN_TO_STUDY]: (studyId, users) => ({
```

Comment:
similar to comment below - but if we are only going to process one user at a time, should we be passing users here?

Reply:
I'm going to bring up during standup tomorrow -- wonder if this makes sense as a follow on.
The current UX is just to do one at a time, and I think my desire to keep this parallel to datasets is getting in the way